yaml/com/ibm/License/LicenseManager.interface.yaml

```diff
@@ -0,0 +1,41 @@
+description: >
+      This Method accepts license key and forwards to host
+properties:
+     - name: LicenseString
+       type: string
+       description: >
+         License string of 34 characters.
+     - name: LicenseActivationStatus
+       type: enum[self.Status]
+       default: Pending
+       description: >
+         License Activation status
+enumerations:
+     - name: Status
+       description: >
+         Status enumerations
+       values:
```

Comment:
Paul okayed this?

Reply:
We get all these back from Phyp ?

Reply:
Doesn't each License have a status ?

E.g. at https://ibm.ent.box.com/file/821056707827?s=htgl7eoo3mviqy527vyxa4c3vrctl3tr 

I see

```
       { 

            "Id": "TrialProcs", 

            "Name": "Trial Processor Licenses", 

            "Type": "Trial", 

            "AuthType": "NumberOfDevice", 

            "AuthDeviceNumber": 8, 

            "Status": "Enabled" 

        }, 
```

Reply:
These are the fileIO response codes for the license upload. These are not the individual license status.

Individual license status are represented by:
xyz.openbmc_project.State.Decorator.Availability
xyz.openbmc_project.State.Decorator.OperationalStatus

They are derived as below
PHYP Enable –> Operational Status true and availability true
PHYP Disable  –> Operational Status false and availability true
Unavailable –> Operational Status false and availability false

Reply:
I think every license should have a status..